recipes-openxt/answerfiles/network-installer-answerfiles_1.0.bb

```diff
@@ -0,0 +1,33 @@
+SUMMARY = "OpenXT installer answerfiles for network boot."
```

Comment:
Delete this file since it should be unused.

Reply:
Yes.
I think we should move the answer files in https://github.com/OpenXT/installer as well, where the other answerfiles are, but since this PR is already dragging along I would like to do it in a different one.

Reply:
Yes, but you already moved the answerfiles into the xenclient-installer do_deploy, so this recipe is unused?

Reply:
Yes, that recipe can go.